internal/cmd/users.go

```diff
@@ -416,6 +437,40 @@ func checkPasswordRequirements(oldPassword string) survey.Validator {
 	}
 }
 
+func checkServerPasswordRequirements(settings *api.PasswordResponse, password string) error {
+	var errs []string
+
+	if !hasMinimumCount(settings.LowercaseMin, password, unicode.IsLower) {
+		errs = append(errs, fmt.Sprintf("- needs minimum %d lower case letters", settings.LowercaseMin))
+	}
+
+	if !hasMinimumCount(settings.UppercaseMin, password, unicode.IsUpper) {
+		errs = append(errs, fmt.Sprintf("- needs minimum %d upper case letters", settings.UppercaseMin))
+	}
+
+	if !hasMinimumCount(settings.NumberMin, password, unicode.IsDigit) {
+		errs = append(errs, fmt.Sprintf("- needs minimum %d numbers", settings.NumberMin))
+	}
+
+	if !hasMinimumCount(settings.SymbolMin, password, isValidSymbol) {
+		errs = append(errs, fmt.Sprintf("- needs minimum %d symbols", settings.SymbolMin))
+	}
+
+	if len(password) < settings.LengthMin {
```

Comment:
This should probably be the first check especially since the output is ordered based on the placement of the check. An alternative is probably to error on first failed check but inform user of the full requirements. Omitting requirements can cause the user to replace valid parts of their input, making them no longer valid.